Attendance_app.py

```diff
@@ -147,32 +146,50 @@ class StartPage(tk.Frame):
+        self.lb_class.place(x=130, y=150)
+        self.tv_class = tk.Entry(self, width=30)
         self.tv_class.focus()
-        self.tv_class.pack()
-        self.lb_username=tk.Label(self,text="USERNAME : ",width=15)
-        self.lb_username.pack()
-        self.tv_username=tk.Entry(self,width=15)
-        self.tv_username.pack()
-        self.lb_pass=tk.Label(self,text="PASSWORD : ",width=15)
-        self.lb_pass.pack()
-        self.tv_pass=tk.Entry(self,show="*",width=15)
-        self.tv_pass.pack()
-
-        button1 = tk.Button(self, text="Go to Student Portal", width=30,
-                            command=lambda: self.doWork("StudentPanelPage",self.tv_class.get(),self.tv_username.get(),self.tv_pass.get()))
-        button2 = tk.Button(self, text="Go to Manager Portal", width=30,
-                            command=lambda: self.doWork("ManagerPanelPage",self.tv_class.get(),self.tv_username.get(),self.tv_pass.get()))
-        button1.pack()
-        button2.pack()
-        bt_exit = tk.Button(self, text="Exit",width=20,
-                           command=self.exit)
-        bt_exit.pack()
+        self.tv_class.place(x=300, y=155)
+
+        self.lb_username = tk.Label(self, text="USERNAME : ", bg=self.bkg, fg=self.text_color, font=("Courier",18))
+        self.lb_username.place(x=130,y=200)
+        self.tv_username = tk.Entry(self,width=30)
+        self.tv_username.place(x=300,y=205)
+
+        self.lb_pass = tk.Label(self, text="PASSWORD : ", bg=self.bkg, fg=self.text_color, font=("Courier",18))
+        self.lb_pass.place(x=130, y=250)
+        self.tv_pass = tk.Entry(self, show="*", width=30)
+        self.tv_pass.place(x=300, y=255)
+        self.showbtn = tk.Button(self, text="SHOW", bg="#ed3833", command=self.show)    #32ff6a
+        self.show = False
+        self.showbtn.place(x=555, y=253)
+
+        self.button1 = tk.Button(self, bg="#45056e", fg=self.text_color, text="Go to Student Portal", width=16,height=3, font=("",15),
+                                 command=lambda: self.doWork("StudentPanelPage",self.tv_class.get(),self.tv_username.get(),self.tv_pass.get()))
+        self.button2 = tk.Button(self, bg="#5f1854", fg=self.text_color, text="Go to Manager Portal", width=16,height=3, font=("",15),
+                                 command=lambda: self.doWork("ManagerPanelPage",self.tv_class.get(),self.tv_username.get(),self.tv_pass.get()))
+        self.button1.place(x=100,y=350)
+        self.button2.place(x=360,y=350)
+        self.bt_exit = tk.Button(self, bg="red", fg="yellow", text="Exit", width=10, command=self.exit)
+        self.bt_exit.place(x=300,y=500)
+
+    def show(self):
+        if self.show==False:
+            self.tv_pass.config(show="")
+            self.show=True
+            self.showbtn.config(bg="#32ff6a")
+        elif self.show==True:
+            self.tv_pass.config(show="*")
+            self.show=False
+            self.showbtn.config(bg="#ed3833")

```

Comment:
Format code according to [PEP8](https://www.python.org/dev/peps/pep-0008/) and [Variable Annotations](https://www.python.org/dev/peps/pep-0008/#variable-annotations).